Description:Remove gemspec git ls files.patch
 Removed call to 'git ls-files' from gem spec.
Subject:Remove gemspec git ls-files
From:Per Andersson <avtobiff@gmail.com>
---
 pygments.rb.gemspec |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/pygments.rb.gemspec
+++ b/pygments.rb.gemspec
@@ -21,5 +21,5 @@
   # s.extensions = ['ext/extconf.rb']
   s.require_paths = ['lib']
 
-  s.files = `git ls-files`.split("\n")
+#  s.files = `git ls-files`.split("\n")
 end
